### PR TITLE
Updated to Node 8.11.1

### PR DESCRIPTION
**Pull request by mcollina**

### CODE REVIEW

**Comment by mcollina** [comment]
CI: https://ci.nodejs.org/view/MyJobs/job/readable-stream-pipeline/13

File: lib/_stream_duplex.js [hunk 1]
```diff
@@ -52,10 +52,13 @@ var Writable = require('./_stream_writable');
 
 util.inherits(Duplex, Readable);
 
-var keys = objectKeys(Writable.prototype);
-for (var v = 0; v < keys.length; v++) {
-  var method = keys[v];
-  if (!Duplex.prototype[method]) Duplex.prototype[method] = Writable.prototype[method];
+{
+  // avoid scope creep, the keys array can then be collected
```
Comment (mafintosh) [inline lib/_stream_duplex.js]: this is now a very confusing comment as we rewrite consts/lets to var, but i guess there is nothing to do about that :joy:

**Comment by mafintosh** [comment]
LTGM on clean ci

**Comment by mafintosh** [comment]
Let's try again, CI: https://ci.nodejs.org/view/MyJobs/job/readable-stream-pipeline/14/

File: lib/_stream_duplex.js (hunk 1 above)
Comment (mcollina) [reply lib/_stream_duplex.js]: all this code is babelified anyway here. So we might even change some of this code in core.

**Comment by mcollina** [comment]
CI is passing, landing.

**Pull request merged**